docs/docs/testing-css-in-js.md

```diff
@@ -35,7 +35,10 @@ In order to tell Jest to use the serializer you'll need to create the file `setu
 
 ```js:title=setup-test-env.js
 import { createSerializer } from "jest-emotion"
-import * as emotion from "emotion"
+// for Emotion 10
+import * as emotion from "@emotion/core"
```

Comment:
I think we'd be good just keeping this line as a change, the comments are accurate but I think that could be found on emotion's docs so we don't have to be responsible for it 🙂

Reply:
Noted. Thanks! I went ahead and updated per your suggestion.